core-web/libs/dojo-theme/scss/dijit/_theme.scss

```diff
@@ -0,0 +1,39 @@
+// BASIC
+@import "variables";
+@import "mixins";
+@import "common";
+@import "icons";
+
+// FORM
+@import "form/common";
+@import "form/textBox";
+@import "form/textArea";
+@import "form/numberSpinner";
+@import "form/button";
+@import "form/checkbox";
+@import "form/radioButton";
+@import "form/select";
+@import "form/combobox";
+@import "form/slider";
+@import "form/timePicker.scss";
+
+// LAYOUT
+@import "layout/accordionContainer";
+@import "layout/borderContainer";
+@import "layout/ContentPane";
+@import "layout/dialogBox";
+@import "layout/popup";
+@import "layout/splitContainer";
+@import "layout/tabContainer";
+@import "layout/tooltip";
```

Comment:
can we correct the filenames because in case-sensitive it's throwing the error.

Reply:
done!